.github/workflows/python-package.yml

```
@@ -12,7 +12,7 @@ jobs:
     - uses: actions/checkout@v3
 
     - name: Set up Python
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v4
```

we could do this and keep the "Install additional Python interpreters" step or we could switch to installing different python releases via a GitHub Actions matrix (like done for liquidsoap versions in `liquidsoap-script.yml`).
```suggestion
      uses: actions/setup-python@v4
      with:
        python-version: '3.x'
```

I'll have to look into how/if that conflicts with tox. Maybe we'd want to just run the tests through pytest on CI and only use/keep tox for local work.